connection.go

```diff
@@ -6,13 +6,17 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/rs/zerolog/log"
+
 	"github.com/databricks/databricks-sql-go/internal/cli_service"
+	"github.com/databricks/databricks-sql-go/internal/client"
+	"github.com/databricks/databricks-sql-go/internal/config"
 	"github.com/databricks/databricks-sql-go/internal/sentinel"
 )
 
 type conn struct {
-	cfg     *config
-	client  *cli_service.TCLIServiceClient
+	cfg     *config.Config
+	client  *client.ThriftServiceClient
```

Comment:
Oh perfect! I was running into the issue of how to mock a concrete implementation (cli_service.TCLIServiceClient) for tests in `connection_test.go` but this will solve the issue

Reply:
Cool